ejabberd 2.1.3 has a module ejabberd_debug
which can be used to gather profiling information
about the running instance of ejabberd.

In upstream source, this module is semi-disabled:
a call to start the server this module implements
is commented out while the call to stop that server is not.
Since we do not package this module, we disable
shutting down the module's server.

Index: pkg-ejabberd/src/ejabberd_app.erl
===================================================================
--- pkg-ejabberd.orig/src/ejabberd_app.erl	2010-05-17 21:52:29.000000000 +0400
+++ pkg-ejabberd/src/ejabberd_app.erl	2010-05-17 21:52:44.000000000 +0400
@@ -85,7 +85,7 @@
 stop(_State) ->
     ?INFO_MSG("ejabberd ~s is stopped in the node ~p", [?VERSION, node()]),
     delete_pid_file(),
-    ejabberd_debug:stop(),
+    %ejabberd_debug:stop(),
     ok.
 
 
